### PR TITLE
GCLOUD2-11797 security group management updated

### PR DESCRIPTION
**Pull request by alexk53**
GCLOUD2-11417 - some fix in documentations, dual-stack public interface example added to gcore_instancev2

### CODE REVIEW

File: gcore/resource_gcore_instancev2.go [hunk 1]
```diff
@@ -337,12 +327,14 @@ For Windows instances, Admin user password is set by 'password' field and cannot
...
+										Type:        schema.TypeString,
+										Required:    true,
+										Description: "IP address",
 									},
 									"type": {
-										Type:     schema.TypeString,
-										Required: true,
+										Type:        schema.TypeString,
+										Required:    true,
+										Description: "Type of the address",
```
[the remark this answers is not in the document]
Comment (alexk53) [reply gcore/resource_gcore_instancev2.go:337]: there is a mistake, it should be computed, not required, i'll fix it

File: docs/resources/instancev2.md [hunk 2]
```diff
@@ -443,29 +474,35 @@ Optional:
...
-<a id="nestedblock--volume"></a>
-### Nested Schema for `volume`
+- `volume_id` (String)
 
 Optional:
 
-- `attachment_tag` (String)
 - `boot_index` (Number) If boot_index==0 volumes can not detached
-- `delete_on_termination` (Boolean)
+- `delete_on_termination` (Boolean) Delete volume on termination
```
[the remark this answers is not in the document]
Comment (alexk53) [reply docs/resources/instancev2.md:487]: Should be read-only, will be fixed

File: docs/resources/instancev2.md [hunk 3]
```diff
@@ -443,29 +474,35 @@ Optional:
...
 
 Optional:
 
-- `attachment_tag` (String)
 - `boot_index` (Number) If boot_index==0 volumes can not detached
-- `delete_on_termination` (Boolean)
+
+Read-Only:
+
+- `attachment_tag` (String) Tag for the volume attachment
```
Comment (IsabellaMerkulova) [inline docs/resources/instancev2.md:490]: I guess you can send this one on volume creation

File: gcore/resource_gcore_instancev2.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (IsabellaMerkulova) [reply gcore/resource_gcore_instancev2.go:339]: Actually I meant make description more wider, but your fix is also good :)

File: docs/resources/instancev2.md (hunk 3 above)
Comment (alexk53) [reply docs/resources/instancev2.md:504]: made it computed

File: docs/resources/instancev2.md [hunk 4]
```diff
@@ -145,7 +154,7 @@ resource "gcore_volume" "boot_volume_windows" {
   region_id  = data.gcore_region.region.id
 }
 
-resource "gcore_instancev2" "instance" {
+resource "gcore_instancev2" "instance-instance" {
```
[the remark this answers is not in the document]
Comment (alexk53) [reply docs/resources/instancev2.md:157]: oops..)

File: docs/resources/instancev2.md [hunk 5]
```diff
@@ -429,43 +474,49 @@ by adding the volume resource inside an instance resource. (see [below for neste
 Required:
 
 - `name` (String) Name of interface, should be unique for the instance
+- `security_groups` (List of String) list of security group IDs, they will be attached to exact interface
 
 Optional:
 
-- `existing_fip_id` (String)
-- `ip_address` (String)
+- `existing_fip_id` (String) The id of the existing floating IP that will be attached to the interface
```
[the remark this answers is not in the document]
Comment (alexk53) [reply docs/resources/instancev2.md:481]: if somebody already manage to create instances with gcore_intancev2 it will broke state

**Review by IsabellaMerkulova — APPROVED** [review]

**Pull request merged**